src/common/libjsc/jstatctl.c

```diff
@@ -309,7 +309,7 @@ static int jobid_exist (flux_t *h, int64_t j)
     if (path == NULL)
         goto done;
     if (!(f = flux_kvs_lookup (h, FLUX_KVS_READDIR, path))
-                                || flux_kvs_lookup_get (f, NULL) < 0) {
+                                || flux_future_get (f, NULL) < 0) {
```

Comment:
this seems like a different cleanup than what this commit is?

Reply:
I'm actually not sure why I did that now.  Will back it out.

Reply:
Oops that change was necessary to avoid breaking git bisect, because in this commit `flux_kvs_lookup_get()` loses the ability to fetch a directory object.

The straightforward change would have been to use `flux_kvs_lookup_get_dir()` and set the 'dir' parameter to NULL.  I'll switch to that.

Testing the future result is just seeing if the RPC returned an error nor not without parsing the directory object, since we only wanted to see if the lookup worked or not here.  Needless optimization.